cmis/pom.xml

```diff
@@ -33,7 +33,8 @@
 			<artifactId>chemistry-opencmis-client-impl</artifactId>
 			<version>${opencmis.version}</version>
 			<exclusions>
-				<exclusion><!-- Exclude 1.0-2 as 1.0.1 is already included by xmlbeans. -->
+				<exclusion>
+					<!-- Exclude 1.0-2 as 1.0.1 is already included by xmlbeans. -->
```

Comment:
Wil je deze pom.xml ook nog effe terugdraaien? Zit geen inhoudelijke wijziging in, zover ik zie.